suites/quincy/rbd/tier-3_rbd_persistent_write_back_cache.yaml

```diff
@@ -491,3 +491,30 @@ tests:
       module: test_rbd_pwl_cache_cluster_operation.py
       name: Validate cluster operation with persistent cache enabled
       polarion-id: CEPH-83574891
+
+  - test:
+      abort-on-fail: true
+      config:
+        levels:
+          - client
+          - pool
+          - image
+        cache_file_size: 1073741824         # 1 GB
+        rbd_persistent_cache_mode: ssd      # "ssd" or "rwl" on pmem device
+        drive: /dev/nvme0n1
+        client: node10
```

Comment:
I only see max 5 node 2 clusters max in the in conf/reef/rbd, so node10 is correct client node ? I see 2 more existing tests using node10. Please check if we are doing right here.

Reply:
for https://github.com/red-hat-storage/cephci/blob/be549b6e960e60b90fee37feedb16d5b5e07a440/suites/quincy/rbd/tier-3_rbd_persistent_write_back_cache.yaml#L423 this recently @manasagowri added these nodes i think it works.